analytics/correlation_category.md

```diff
@@ -0,0 +1,35 @@
+# Correlation by Category
+
+Calculates Correlation by Category.
+
+## Input Data
+Input data should contain categorical columns and numeric columns.
+
+  * Category Column - Categorical column that you want to see correlation.
```

Comment:
Categorical column that defines the groups among which correlations are calculated.

Reply:
Changed as suggested.